src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/ProjectAnalyzerReference.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.MSBuild
+{
+    /// <summary>
+    /// Represents a reference to another project file.
```

Comment:
This description seems confusing -- this is just a marshallable AnalyzerReference or whatever we call those? What is path a path too?